pages/quiz/about-you.tsx

```diff
@@ -1,26 +1,34 @@
 import React, { useContext } from 'react'
 import styled from 'styled-components'
 import Link from 'next/link'
+import { useRouter } from 'next/router'
 
 import { Layout } from '../../src/Components/Layout'
 import { HeaderComponent } from '../../src/Components/Header'
 import { VerticalSpacing } from '../../src/Components/VerticalSpacing'
 import { Checkbox } from '../../src/Components/Checkbox'
 import { RadioButton } from '../../src/Components/RadioButton'
 import { QuizContext } from '../../src/context/QuizContext'
+import { LinkButton } from '../../src/Components/LinkButton'
 
 const Section = styled.section`
   align-items: center;
   display: flex;
   flex-direction: column;
   justify-content: space-between;
 `
+const Navigation = styled.section`
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  padding: 1rem;
+  width: 100%;
+`
```

Comment:
Please can we make this a shared component instead of duplicating it across each file? Thanks.